time/src/main/java/net/jqwik/time/api/DateTimes.java

```diff
@@ -58,9 +58,9 @@ public static OffsetDateTimeArbitrary offsetDateTimes() {
 	 *
 	 * @return a new arbitrary instance
 	 */
-	/*public static ZonedDateTimeArbitrary zonedDateTime() {
-		return null; TODO
-	}*/
+	public static ZonedDateTimeArbitrary zonedDateTime() {
```

Comment:
> Would it be possible to use ZonedDateTime or maybe Instant values instead?

Maybe instant makes sense, but ZonedDateTime not. In the time-based arbitraries we used LocalTime (OffsetTime, ...)

What do you think is better? Instant or LocalDateTime?

Reply:
IMO Instant makes sense because it represents an absolute point in time. But that means that the semantics change: The resulting ZonedDateTime or OffsetDateTime must fulfill the constraints when they are converted to Instant.

Reply:
Alternatively we could stick with LocalDateTime and the current semanctics, but then this has to be documented in the Javadoc. That's probably less effort and I'm also fine with it.

Reply:
I'll think about it